packages/styled-components/src/components/primitives/Form/CurrencyField/CurrencyField.test.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
```

Comment:
Could we rewrite these tests to look like `ExpandableTextfield` tests? We have a lot of repeated code

Reply:
done!